src/components/map/layers/NodeLayer.tsx

```diff
@@ -57,6 +64,17 @@ export default class NodeLayer extends Component<MarkerLayerProps> {
         />`;
     }
 
+    private getUnknownNodeMarkerHtml = (isSelected: boolean) => {
+        const borderColor = isSelected ?
+            color.UNKNOWN_BORDER_COLOR_SELECTED : color.UNKNOWN_BORDER_COLOR;
+        const fillColor = isSelected ?
+            color.UNKNOWN_FILL_COLOR_SELECTED : color.UNKNOWN_FILL_COLOR;
+        return `<div
+            style="border-color: ${borderColor}; background-color: ${fillColor}"
+            class=${s.nodeContent}
```

Comment:
I see this inline html javascript 3 x times ---> make a function to say it only once

Reply:
Also hmm, we could just use class to declare all of this just like in crossroadCircle and stopCircle.

`<div class=${s.nodeContent s.unknownNode}></div>`

.unknownNode {
   color: ...
   fillColor: ... or is it just fill: ?
}

So instead of saying:
html = this.getUnknownNodeMarkerHtml(isSelected);

you should say:
html = `<div class=${s.nodeContent s.unknownNode}></div>`